services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/FlowOperationsBolt.java

```diff
@@ -79,9 +88,31 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
                 .collect(Collectors.toList());
     }
 
+    private List<FlowsResponse> processRerouteFlowsForLinkRequest(RerouteFlowsForIslRequest message, Tuple tuple)
+            throws IslNotFoundException {
+        SwitchId srcSwitch = message.getSource().getDatapath();
+        Integer srcPort = message.getSource().getPortNumber();
+        SwitchId dstSwitch = message.getDestination().getDatapath();
+        Integer dstPort = message.getDestination().getPortNumber();
+
+        List<String> flowIds = flowOperationsService.getFlowIdsForLink(srcSwitch, srcPort, dstSwitch, dstPort).stream()
+                .map(FlowPair::getForward)
+                .map(Flow::getFlowId)
+                .collect(Collectors.toList());
+
+        flowIds.forEach(flowId -> {
```

Comment:
No needs to create collection with flowIds, you may process them with forEach on Stream

Reply:
This collection is a return value.